scss/_patterns_navigation.scss

```diff
@@ -249,8 +250,6 @@ $spv-navigation-logo: 0.3125rem;
     .p-navigation__link {
       @extend %navigation-link;
 
-      padding-left: $navigation-logo-tag-width + $sph-navigation-link;
```

Comment:
The padding should be set here instead of line 654, so it affects only tagged logo, not standard one.

And it would be in one place instead of 2 elements.

![padding](https://user-images.githubusercontent.com/83575/196949935-5d19ba83-6391-4b1f-a04a-45833b173cc3.gif)